src/components/StatusStrip/_status-strip.scss

```diff
@@ -0,0 +1,13 @@
+.status-strip {
+  align-items: center;
+  display: flex;
+  font-size: 0.875rem;
```

Comment:
Should use `@extend %small-text;` to ensure the component has the right line-height, etc.

Reply:
<img width="862" alt="Screenshot 2020-11-25 at 20 31 40" src="https://user-images.githubusercontent.com/505570/100278726-415cd900-2f5d-11eb-9e4f-a536022dd5dd.png">

`%small-text` is not exposed by Vanilla?

Reply:
It is via the `vf-b-placeholders` mixin. Which is a [requirement when import a custom Vanilla](https://vanillaframework.io/docs/customising-vanilla#importing-individual-components). Or you bite the bullet and import the entire Vanilla. After testing it adds 9kb to the styles in the application.

Reply:
Is that 9kb for the whole of vanilla? Or just the mixin?

Reply:
It's time to import the lot.

Reply:
The entire framework. At this point your are including almost all of it individually.